stack/stack.go

```diff
@@ -2,115 +2,142 @@ package stack
 
 import (
 	"encoding/json"
+	"errors"
 	"github.com/Appliscale/perun/context"
 	"github.com/Appliscale/perun/mysession"
 	"github.com/Appliscale/perun/parameters"
 	"github.com/Appliscale/perun/progress"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/cloudformation"
 	"io/ioutil"
+	"path"
 )
 
+// This function reads "StackName" from Stack in CliArguments and file from TemplatePath in CliArguments. It converts these to type string.
+func getTemplateFromFile(context *context.Context) (string, string, error) {
+	var rawTemplate []byte
+	var readFileError error
+	path, pathError := getPath(context)
+	if pathError != nil {
+		return "", "", pathError
+	}
+
+	rawTemplate, readFileError = ioutil.ReadFile(path)
+	if readFileError != nil {
+		context.Logger.Error(readFileError.Error())
+		return "", "", readFileError
+	}
+
+	rawStackName := *context.CliArguments.Stack
+	template := string(rawTemplate)
+	stackName := rawStackName
+	return template, stackName, nil
+}
+
+func getPath(context *context.Context) (path string, err error) {
+	if *context.CliArguments.Mode == "create-stack" {
+		path = *context.CliArguments.TemplatePath
+	} else if *context.CliArguments.Mode == "set-stack-policy" {
+		if *context.CliArguments.Unblock {
+			path = "./stack/defaultstackpolicy/unblocked.json"
```

Comment:
This can't work this way, see above (about binary distribution). The solution must work similarly to the configuration files (ideally in `/etc` or user home directory). 

We need to modify also `Makefile` to copy those and distribute both with binaries when releasing. I believe @rusty-2 works on something similar regarding heuristics file.

@pfigwer What do you think about it?